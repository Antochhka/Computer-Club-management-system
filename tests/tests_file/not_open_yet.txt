2
10:00 20:00
15
09:30 1 client1
10:00 1 client2