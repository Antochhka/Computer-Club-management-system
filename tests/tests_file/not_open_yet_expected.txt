10:00
09:30 1 client1
09:30 13 NotOpenYet
10:00 1 client2
20:00 11 client2
20:00
1 0 00:00
2 0 00:00